### PR TITLE
Configuring multiclass mask

### PR DESCRIPTION
**Pull request by nthndy**
I've added extra mask channels, which is important for the latest commits i pushed to the multiclass object tracking, and I've (temporarily) commented out some transform function aspects as they were redundant with the .npy transforms I was using. I think going forward we should decide on a way to maybe load different classes of masks as one channel and also decide on a common transform input type, or expand it to accept different types.

### CODE REVIEW

File: octopuslite/transform.py [hunk 1]
```diff
@@ -17,21 +17,25 @@ def __call__(self, x, idx):
...
     Parameters
     ----------
     path : PathLike
         A path to either a numpy file of transforms, or a folder containing
         transforms as XML files.
-    n : int
-        The number of frames in the movie.
+    # # may become necessary when alternate transform format is used
+    # n : int
+    #     The number of frames in the movie.
```
Comment (quantumjot) [inline octopuslite/transform.py:38]: Rather than comment out `n` can you just remove it?

File: README.md [hunk 2]
```diff
@@ -9,7 +9,22 @@ address different channels using the `Channels` enumerator.
...
+Parameters
+  ----------
+  path : str
+      The path to the dataset.
+  crop : tuple, optional
+      An optional tuple which can be used to perform a centred crop on the data.
+  transforms : Path or np.ndarray
+      Transforms to be applied to the image stack.
+  remove_background : bool
+      Use a estimated polynomial surface to remove uneven illumination.
```
Comment (quantumjot) [inline README.md:29]:
```suggestion
  Parameters
  ----------
  path : str
      The path to the dataset.
  crop : tuple, optional
      An optional tuple which can be used to perform a centred crop on the data.
  transforms : Path or np.ndarray
      Transforms to be applied to the image stack.
  remove_background : bool
      Use a estimated polynomial surface to remove uneven illumination.
```
```suggestion
Parameters
  ----------
  path : str
      The path to the dataset.
  crop : tuple, optional
      An optional tuple which can be used to perform a centred crop on the data.
  transforms : Path or np.ndarray
      Transforms to be applied to the image stack.
  remove_background : bool
      Use a estimated polynomial surface to remove uneven illumination.
```

File: octopuslite/utils.py [hunk 3]
```diff
@@ -20,7 +20,12 @@ class Channels(enum.Enum):
     RFP = 2
     IRFP = 3
     PHASE = 4
-    WEIGHTS = 98
+
```
Comment (quantumjot) [inline octopuslite/utils.py:23]:
```suggestion
```

File: octopuslite/utils.py [hunk 4]
```diff
@@ -20,7 +20,12 @@ class Channels(enum.Enum):
...
+    WEIGHTS = 50
+
```
Comment (quantumjot) [inline octopuslite/utils.py:25]:
```suggestion
```

**Review by quantumjot — CHANGES_REQUESTED** [review]
Looks good - can you make these few changes?

File: octopuslite/transform.py [hunk 5]
```diff
@@ -17,21 +17,25 @@ def __call__(self, x, idx):
         if self.transforms is None:
             return x
 
-        tform = tf.AffineTransform(translation=self.transforms[idx, ...])
+        #tform = tf.AffineTransform(translation=self.transforms[idx, ...])
+        tform = tf.AffineTransform(translation=self.transforms[idx, :2, 2])
```
[the remark this answers is not in the document]
Comment (nthndy) [reply octopuslite/transform.py:21]: yeah but by specifying the parameter `translation=...` it achieves the same thing

File: octopuslite/transform.py (hunk 1 above)
Comment (nthndy) [reply octopuslite/transform.py]: done ✅

**Review by quantumjot — APPROVED** [review]
Looks good!

**Comment by quantumjot** [comment]
BTW - @nthndy the tests are failing because of the code formatting issues (see #14) - you may want to configure pre-commit for future PRs

**Comment by nthndy** [comment]
yes i been meaning to do this, will do now

**Pull request merged**